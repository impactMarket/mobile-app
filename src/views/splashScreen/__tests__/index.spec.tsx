import React from 'react';
import { create } from 'react-test-renderer';

import SplashScreen from '../index';

const tree = create(<SplashScreen />).toJSON();

describe('Splash screen test', () => {
    test('should render SplashScreen correctly', () => {
        expect(tree).toMatchSnapshot();
    });
});
