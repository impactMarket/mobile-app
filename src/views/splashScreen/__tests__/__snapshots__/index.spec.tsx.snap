// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash screen test suite should render Splash screen correctly 1`] = `ShallowWrapper {}`;
